utils/towers.ahk

```diff
@@ -29,7 +29,7 @@ Targeting(tower, tabCount) {
     Sleep(200)
 }
 
-Upgrade(tower, topCount, middleCount, bottomCount) {
+Upgrade(tower, topCount, middleCount, bottomCount, option := "") {
```

Comment:
The new parameter should probably just be a boolean called `asap` with the default value of `false` as I don't really see many other options than asap that would be useful.

This would also simplify `if option = "asap" {` to just `if asap {`